thirdeye/thirdeye-frontend/app/styles/shared/_styles.scss

```diff
@@ -808,7 +808,8 @@ body {
   &__list {
     list-style: none;
     font-weight: normal;
-    margin-bottom: 0;
+    //select 17px as property value from vision
```

Comment:
What exactly does 'vision' refer to?

Reply:
It refers to the UI page. The value is selected by observing the UI component on the reality page. The `17px` is an Approximate value. Let me know whether you have a good suggestion for the explanation if it's confusing for you.